projects/observability/src/pages/saved-queries/saved-queries.component.ts

```diff
@@ -0,0 +1,52 @@
+import { ChangeDetectionStrategy, Component, OnDestroy } from '@angular/core';
+import { Subscription } from 'rxjs';
+
+import { NavigationService, PreferenceService } from '@hypertrace/common';
+import { DrilldownFilter, ExplorerService } from '../explorer/explorer-service';
+import { SavedQuery } from '../explorer/explorer.component';
+
+@Component({
+  styleUrls: ['./saved-queries.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  template: `
+    <div class="saved-queries">
+      <ht-page-header class="explorer-header"></ht-page-header>
+      <div class="query-list-container">
+        <a *ngFor="let query of savedQueries" (click)="onClick(query)">
+          <span *ngFor="let filter of query.filters">{{ filter.userString }}</span>
+        </a>
+      </div>
+      <p class="not-found-text" *ngIf="savedQueries.length === 0">
+        You haven't saved any queries! Go to Explorer page to save a query.
+      </p>
+    </div>
```

Comment:
More of a feature request for future - Ability to selectively remove a saved query. Just thought to mention this, so development is done keeping that in mind,

Reply:
Yes, deleting a query is an important feature. I'm just not sure how to represent the delete button in the UI. I'm thinking maybe a kebab menu on the top right of each query like GitHub does:

<img width="350" alt="Screenshot 2022-05-23 at 10 49 51 AM" src="https://user-images.githubusercontent.com/29686866/169748865-e717544c-e88c-41f1-b375-f10414f564ae.png">

But I don't Hypertrace has any UI element like this. Would need to create one. What do you think?

We can discuss this further here:
https://razorpay.atlassian.net/browse/OBSV-726

Reply:
Will have further discussions in the issue you have created, resolving this comment for now.